roles/rook/README.rst

```diff
@@ -0,0 +1,101 @@
+Rook
+========
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+rook-jenkins-update - Updating the jenkins app to the version that is defined in the "jenkins_master_image" varible
+rook-os-update - Updating the packages of the rook jenkins OS
+rook-recovery - Recoverying the Prod-jenkins from the image defined in the "image" varible in a case that the instance was deleted
+
+Usage
++++++
+
+The rook role is primarily used by the ``rook.yml`` playbook.  Run this playbook with one of the optional tags listed in the ``rook.yml``
+to upgrade rook jenkins os/app or recoverying the instance from latest image
+
+**NOTE:** ``rook.yml`` Is currently using only localhost and not any host from the inventory in the inventory this is because the rook-recovery creates and configures the rook jenkins during his run.
+**NOTE:** After running play with the rook-recovery tag you will need to make the new recoverd instance available the to public network once the instance is deployed, by adding it to the load balancing target group under the rook ci vpc access details to the aws dashboard can be found in here http://wiki.ceph.redhat.com/dokuwiki/doku.php?id=rook_aws_account
+
+Examples
++++++
+
+Updating the rook jenkins app to version 2.289.1
+ansible-playbook rook.yml --tags="rook-jenkins-update" --extra-vars="jenkins_master_image=jenkins/jenkins:2.289.1"
```

Comment:
Format this like the `Variables` below.

e.g.,
```
Updating the rook jenkins app to version 2.289.1::

    ansible-playbook rook.yml --tags="rook-jenkins-update" --extra-vars="jenkins_master_image=jenkins/jenkins:2.289.1"

```

Reply:
Updated the readme file